src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1097,8 +1099,8 @@ private void logFlush(String flushReason)
             }
         }
 
-        logger.debug("Enqueuing flush of {} for cause {}: {}", name, flushReason, String.format("%d (%.0f%%) on-heap, %d (%.0f%%) off-heap",
-                                                                     onHeapTotal, onHeapRatio * 100, offHeapTotal, offHeapRatio * 100));
+        logger.debug("Enqueuing flush of {} for cause {}: {}", SafeArg.of("cfName", name), SafeArg.of("flushCause", flushReason), SafeArg.of("heapInfo", String.format("%d (%.0f%%) on-heap, %d (%.0f%%) off-heap",
+                onHeapTotal, onHeapRatio * 100, offHeapTotal, offHeapRatio * 100)));
```

Comment:
Should we update this to explicitly log these as params instead of having to parse out the components downstream?
```suggestion
        logger.debug("Enqueuing flush of {} for cause {}: {}", SafeArg.of("cfName", name), SafeArg.of("flushCause", flushReason), SafeArg.of("onHeapTotal", onHeapTotal), SafeArg.of("onHeapPercent", onHeapRatio * 100), SafeArg.of("offHeapTotal", offHeapTotal), SafeArg.of("offHeapPercent", offHeapRatio * 100));
```